### ISSUE TITLE
String live_map_button reference not working

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#12725]
Crowdin is not clever enough for this, so I set the string to be hidden from translation:
![image](https://user-images.githubusercontent.com/949669/153710007-4764feb4-d80c-49aa-a682-6d4a75ae616d.png)

_Originally posted by @Lineflyer in https://github.com/cgeo/cgeo/issues/12715#issuecomment-1037172779_

What I did not take into account:
Hiding this string from translation will remove it in the translated strings, which means it will fall back to the english string for all languages. 
You can see this in PR https://github.com/cgeo/cgeo/pull/12723

@moving-bits: Seems we need to refrain from using such linkage.

**Comment by Lineflyer** [comment]
Best solution seems to be to implement to directly use `map_map` for `release` and clean it up on `master` again to use  `live_map_button` with new content `Map`.

**Comment by fm-sys** [comment]
Please take a look at #12726 as quick-fix...

**Comment by moving-bits** [comment]
> What I did not take into account: Hiding this string from translation will remove it in the translated strings, which means it will fall back to the english string for all languages.

Yes, it takes the English definition for `live_map_button`, recognizes it as a reference to `map_map`, and takes the _translated_ version of that resource. Isn't that exactly what is intended here? (Also see https://github.com/cgeo/cgeo/pull/12722#issuecomment-1037369655)

**Comment by moving-bits** [comment]
> I would still like to avoid such references also to avoid manual steps on Crowdin.

Ok, understandable

> We could keep in the reference on release (as it works) and clean up on `master`. How about that?

Either way would be ok for me. If we have the time for outstanding translations on `release` (which we probably should calculate anyway, as there is a new string "Offline map"), then it does probably make sense to do it the same way on both branches. We could take #12726 and just remove renaming of shortcuts, for example.
